src/Gui/DlgSettings3DView.ui

```diff
@@ -292,14 +292,58 @@ but slower response to any scene changes.</string>
           </property>
          </widget>
         </item>
-        <item row="4" column="0">
+         <item row="4" column="0">
+           <widget class="QLabel" name="spinPickRadiusLabel">
+             <property name="text">
+               <string>Pick radius (px):</string>
+             </property>
+           </widget>
+         </item>
+         <item row="4" column="4">
+           <widget class="Gui::PrefDoubleSpinBox" name="spinPickRadius">
+             <property name="minimumSize">
+               <size>
+                 <width>120</width>
+                 <height>0</height>
+               </size>
+             </property>
+             <property name="toolTip">
+               <string>
+                 Area for picking elements in 3D view.
+                 Larger value eases to pick things, but can make small features impossible to select.
```

Comment:
how does this tooltip look with all the extra space included on line 313 ?

Reply:
good catch, I will resolve it as soon as possible